code/datums/effects/bleeding.dm

```diff
@@ -111,7 +112,9 @@
 
 		var/mob/living/carbon/human/affected_human = affected_mob
 		if(istype(affected_human))
+			///Changed CHEM_EFFECT_NO_BLEEDING effect to also cure bleeding via blood_loss reduction. Preferably this would be modified by the property level
```

Comment:
```suggestion
			//CHEM_EFFECT_NO_BLEEDING effect cure bleeding via blood_loss reduction. Preferably this would be modified by the property level
```